src/services/autoprojection.js

```diff
@@ -0,0 +1,85 @@
+goog.provide('ngeo.AutoProjection');
+
+goog.require('ngeo');
+goog.require('ol.extent');
+goog.require('ol.proj');
+
+/**
+ * TODO
+ * @constructor
+ * @ngdoc service
+ * @ngname ngeoAutoProjection
+ */
+ngeo.AutoProjection = function() {};
+
+
+/**
+ * TODO
+ * @param {string} str TODO
+ * @return {?ol.Coordinate} TODO
+ * @export
+ */
+ngeo.AutoProjection.prototype.stringToCoordinates = function(str) {
```

Comment:
If str is `' '` I can't predict that the code will do...
I think that if one of the `parseFloat` return `NaN`, this function should return `null`.